zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskQueryController.java

```diff
@@ -78,4 +82,27 @@ public ResponseEntity<Object> getByKey(@PathVariable("userTaskKey") final Long u
       return RestErrorMapper.mapProblemToResponse(problemDetail);
     }
   }
+
+  @GetMapping(
+      path = "/{userTaskKey}/form",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE})
+  public ResponseEntity<Object> getFormByUserTaskKey(
+      @PathVariable("userTaskKey") final Long userTaskKey) {
+    try {
+      final Long formKey = userTaskServices.getByKey(userTaskKey).formKey();
+
+      if (formKey == null) {
+        throw new NotFoundException(
```

Comment:
I wonder if we should rather return HTTP 204, to indicate that the request was processed successfully, but there is no form to return.
I think we should reserve the 404 to when:
- the task with the key provided by the caller does not exist.
- the task has a formKey but the form cannot be found